wp-content/themes/redmond/views/layout/footer.html.twig

```diff
@@ -1,3 +1,37 @@
 <footer class="footer">
-
-</footer>
\ No newline at end of file
+    <div class="row top-row">
```

Comment:
make the top-row a parent div with row inside it.  Generally i make the select able div the parent of the row instead of the row in case we need to tweak spacing or something else without conflicting with the row styles.

also when possible use more meaningful names. Instead of top row maybe call this footer-icons